Layout/UIView+Layout.swift

```diff
@@ -278,6 +278,19 @@ extension UIView: LayoutManaged {
         }
     }
 
+	// Set expression value from string
+	@objc open func setStringValue(_ value: String, forExpression name: String) throws {
+		if let typ = type(of: self).cachedExpressionTypes[name] {
+			if let node = _layoutNode {
+				if let exp = LayoutExpression(expression: value, type: typ, for:node) {
+					if let res = try? exp.evaluate() {
```

Comment:
The setStringValue is marked as throwing, but all the errors are suppressed using `try?` - was that intentional?

Reply:
I initially copied the method signature of the other setValue, and so kept the throwing attribute.
For my project, I didn't really care about any error - set it or fail silently.
For the general use case, maybe it should be best for the errors to bubble up. (Although I don't know what can be done with the error, outside of this class... Maybe logging it?)

Reply:
OK, I thought about it and it should be good to bubble up errors that happen during expression evaluation and newvalue setting.